### PR TITLE
Feature company

### PR DESCRIPTION
**Pull request by thordurh**
added some company content, on hold until final design for the page is in.

### CODE REVIEW

File: components/CompanyNav/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  return (
+    <div className={styles.footerBoxContainer}>
+      <div className={styles.footerBox}>
+        {companyData.map((postDetail) => {
+          return (
+            <div>
+              <div className={styles.linkstyle}>
+                <Label styler='footerLabel' text={postDetail.title} htmlFor='companyContent' />
+              </div>
+              <CompanyContent data={postDetail} />
```
Comment (audunnhrafn) [inline components/CompanyNav/index.js]: damn you are nice

File: pages/company/icelandaircargo/index.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import React from 'react'
+import CompanyNav from 'components/CompanyNav'
+import styles from './assets/index.scss'
+
+function Index () {
+  return (
+    <div className={styles.recent}>
+      <CompanyNav />
+      <p>
+        Since overseas flights from Iceland began, air cargo has played a vital role in the Icelandic aviation industry. Both Loftleiðir and Flugfélag Íslands carried cargo on their commercial flights and with the founding of Icelandair in 1973, a Cargo Department was established. Cargo transportation became an important part of the operation and in addition to offering belly space on Icelandair's passenger flights, a freighter flew weekly to Ostende, Belgium. In 1997 the company started Boeing 737 freighter flights to Cologne in Germany, 6 days a week. Demand soon exceeded supply and in September 1999 Boeing 757 freighter flights to Liege in Belgium and to New York commenced. <br /><br />
```
Comment (audunnhrafn) [inline pages/company/icelandaircargo/index.js]: Escape '

File: pages/company/index.js [hunk 3]
```diff
@@ -1,10 +1,12 @@
 import React from 'react'
+import CompanyNav from 'components/CompanyNav'
 import styles from './assets/index.scss'
 
 function Index () {
```
Comment (audunnhrafn) [inline pages/company/index.js]: This does not need to be a function

File: pages/company/recent/index.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+import CompanyNav from 'components/CompanyNav'
+import styles from './assets/index.scss'
+
+function Index () {
+  return (
+    <div className={styles.recent}>
+      <CompanyNav />
+      <p>
+      Icelandair Group's business concept is built on Icelandair's route network and on marketing Iceland as a year-round destination. In addition to the international passenger flights operated by Icelandair, the Group has extensive interests in most other parts of Icelandic tourism and aviation.<br /><br />
+      4000 Engaged and skilled employees with the male/female ratio of 44/56 are the foundation of Icelandair Group's quality service, satisfied customers, positive image, and good financial performance. Environmental responsibility and dedication to minimising the environmental impact of our operations are guiding principles at Icelandair Group.<br /><br />
```
Comment (audunnhrafn) [inline pages/company/recent/index.js]: Escape '

File: pages/company/index.js (hunk 3 above)
Comment (thordurh) [reply pages/company/index.js]: fixed

File: components/CompanyNav/index.js (hunk 1 above)
Comment (thordurh) [reply components/CompanyNav/index.js]: takk kútur

**Pull request merged**